cfme/base/ui.py

```diff
@@ -63,20 +63,6 @@ def address(self):
     return self.appliance.url
 
 
-def prepare_server_name(name):
```

Comment:
I am currently trying to deal with this in
https://github.com/ManageIQ/integration_tests/pull/8620/commits/fc657e8b8ec3b275fa47222a1ce25886ae5b5d29
and the parent patch.

Reply:
The prepare_server_name function was introduced as a workaround for the issue in BZ 1668849, so now that the bug is fixed, I don't see any need to keep it.